packages/menu/src/Menu.ts

```diff
@@ -925,6 +980,14 @@ export class Menu extends SizedMixin(SpectrumElement, { noDefaultSize: true }) {
     }
 
     public override disconnectedCallback(): void {
+        // Clean up touch event listeners to prevent memory leaks
```

Comment:
shouldnt we remove all event listeners then not just iPad?

Reply:
If we only want these event listeners to exist while attached to the DOM and to remove them during the `disconnected` phase of the lifecycle, then we should have been adding them on the `connected` part of the lifecycle instead of the `constructor`.

Right now, if you were to remove the picker from the DOM and add it back, the listeners would not be re-applied.